controller/metrics/metrics.go

```diff
@@ -381,7 +381,11 @@ func (c *appCollector) collectApps(ch chan<- prometheus.Metric, app *argoappv1.A
 		healthStatus = health.HealthStatusUnknown
 	}
 
-	addGauge(descAppInfo, 1, git.NormalizeGitURL(app.Spec.GetSource().RepoURL), app.Spec.Destination.Server, app.Spec.Destination.Namespace, string(syncStatus), string(healthStatus), operation)
+	autoSyncEnabled := false
+
+	autoSyncEnabled = app.Spec.SyncPolicy != nil && app.Spec.SyncPolicy.Automated != nil
```

Comment:
nit: we can replace this 2 lines with below:

```suggestion

	autoSyncEnabled := app.Spec.SyncPolicy != nil && app.Spec.SyncPolicy.Automated != nil
```

Reply:
Appreciate the feedback! Pushed a commit